### PR TITLE
Add Sergiu Szatmari to README contributors list

### PR DESCRIPTION
**Pull request by sergiu-szatmari**
Follow-up to #629

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,6 +142,8 @@ This project wouldn't be possible without our awesome contributors. Thank you >:
             <a href="https://www.facebook.com/melnic.ira/"><img src="https://raw.githubusercontent.com/FrontEnd-ro/frontend.ro/master/client/public/images/contributors/iramelnic.jpg" width="120px;" alt="Ira Melnic"/><br /><sub><b>Ira Melnic</b></sub></a></td>
         <td align="center">
             <a href="https://lazarnarcis.github.io/"><img src="https://raw.githubusercontent.com/FrontEnd-ro/frontend.ro/master/client/public/images/contributors/lazarnarcis.png" width="120px;" alt="Narcis Lazar"/><br /><sub><b>Narcis Lazar</b></sub></a></td>
+        <td align="center">
+            <a href="https://sergiu-szatmari.github.io/"><img src="https://raw.githubusercontent.com/FrontEnd-ro/frontend.ro/master/client/public/images/contributors/sergiu-sz.png" width="120px;" alt="Sergiu Szatmari"/><br /><sub><b>Sergiu Szatmari</b></sub></a></td>
```
Comment (iampava) [inline README.md:146]: A couple of small things:

1. As mentioned in [CONTRIBUTING.md](https://github.com/FrontEnd-ro/frontend.ro/blob/master/CONTRIBUTING.md#commit-messages) - every commit must have an issue associated. So please create an issue in the repo regarding this task
2. Please rename the photo according to your full github username. This is the pattern we're using
3. Squash all commits into one. I will merge after that :D

**Review by iampava — CHANGES_REQUESTED** [review]

**Review by iampava — CHANGES_REQUESTED** [review]
I think you didn't correctly fetch the changes from master.

It looks like you have 20 files chaged...
image

**Comment by sergiu-szatmari** [comment]
@iampava i did fetch the upstream changes but because of the squash Github everything that the squash commit contained as a change from me :laughing: . Should be good now. Can't you also merge as squash commit frum GitHub console? Just like in BitBucket? :thinking:

**Comment by iampava** [comment]
Hey @sergiu-szatmari! There are still 2 commits here 😬

Regarding me squashing and merging, I think it might be possible but honestly never done it :D

**Comment by sergiu-szatmari** [comment]
@iampava this time should be good for real :grin: however, i'd like to suggest the approach in the picture for future mergings, this might be a way of making sure that the squashing was done correctly.

Screenshot from 2022-03-20 16-49-36

Picture taken from [this](https://www.git-tower.com/learn/git/faq/git-squash) article

**Review by iampava — APPROVED** [review]

**Pull request merged**